lib/browser/init.js

```diff
@@ -103,14 +104,14 @@ require('./guest-window-manager')
 // Now we try to load app's package.json.
 let packagePath = null
 let packageJson = null
-const searchPaths = ['app', 'app.asar', 'default_app.asar']
+const searchPaths = asar.getSearchPath()
```

Comment:
Why not just `['app.asar']`?

Reply:
This file is not part of the binary (like "asar.js"). So you're probably able to modify the "electron.asar", where the "searchPaths" can be found in plaintext (if you do so).

Therefore, the "getSearchPaths" is required (see https://github.com/procilon/electron/pull/2/files#diff-30497f0d340d872cd5c3c0d6a87cfa0bR161). 

Proof, that "init.js" lies in electron.asar:
```bash
➜  Resources git:(master) pwd
/...some-electron-project.../dist/mac/App.app/Contents/Resources
➜  Resources git:(master) cat electron.asar | grep "const searchPaths = "
Binary file (standard input) matches
```

Reply:
we will investigate further. But as long as the code in electron.asar is loaded with integrity verification js code is just fine.